### PR TITLE
Fixes #18483: Documentation for /nodes/status page

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/18483

### CODE REVIEW

**Comment by amousset** [comment]
Looks good but cocs are missing :smile:

**Comment by fanf** [comment]
PR updated with a new commit

File: webapp/sources/api-doc/paths/nodes/status.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                type: string
+                description: Result of the request
+                enum:
+                  - success
+                  - error
+              action:
+                type: string
+                description: The id of the action
+                enum:
+                  - nodeDetails
```
Comment (amousset) [inline webapp/sources/api-doc/paths/nodes/status.yml:38]:
```suggestion
                  - getNodesStatus
```

**Comment by fanf** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**